app/konnect/advanced-analytics/llm-reporting.md

```diff
@@ -0,0 +1,118 @@
+|----------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Completion Tokens     | Count         | Completion tokens are any tokens that the model generates in response to an input.       |
+| Prompt Tokens         | Count         | Prompt tokens are the tokens input into the model.            |
+| Total Tokens          | Count         | Sum of all tokens used in a single request to the model. It includes both the tokens in the input (prompt) and the tokens generated by the model (completion).  |
+| Time per Tokens       | Number  | Average time in milliseconds to generate a token. Calculated as LLM latency divided by the number of tokens.                                                       |
+| Costs                 | Cost  | Represents the resulting costs for a request. Final costs = (total number of prompt tokens × input cost per token) + (total number of completion tokens × output cost per token) + (total number of prompt tokens × embedding cost per token). |
+| Response Model        | String        | Represents which AI model was used to process the prompt by the AI provider.   |
+| Request Model         | String        | Represents which AI model was used to process the prompt. |
+| Provider Name         | String        | Represents which AI provider was used to process the prompt.   |
+| Plugin ID             | String        | Represents the UUID of the plugin configuration.          |
+| LLM Latency           | Latency  | Total time taken to receive a full response after a request sent from Kong (LLM latency + connection time).             |
+| Embeddings Latency    | Latency  | Time taken to generate the vector for the prompt string.  |
+| Fetch Latency         | Latency | Total time taken to return a cache.                          |
+| Cache Status          | String        | Shows if the response comes directly from the upstream or not. Possible values: `hit` or `Miss`.                                          |
+| Embeddings Model      | String        | AI providers may have multiple embedding models. This represents the model used for the embeddings.                      |
+| Embeddings Provider   | String        | Provider used for generating embeddings.                                                            |
+| Embeddings Token      | Count         | Tokens input into the model for embeddings.  |
+| Embeddings Cost       | Cost  | Cost of caching.  |
+| Cost Savings          | Cost  | Cost savings from cache.            |
```

Comment:
Nice to have: I'd like more info about how the the cost savings from the cache is calculated.